### PR TITLE
API Reference update to cce-release-v4-oct24

### PR DESCRIPTION
**Pull request by undavide**
# ⚠️ Just a placeholder PR for the time being

Please check it [on Stage](https://developer-stage.adobe.com/express/embed-sdk/docs/v4/)

## Description

Update the API Reference to SDK v4.16.11
Update the changelog

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/changelog/index.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ contributors:
 
 # Changelog
 
+## [4.16.11] - 2024-11-18
+
+### Enhancements
+
+- [`CCEverywhere.appConfig.imageStyleReference`](../../v4/shared/src/types/module/AppConfig.types/interfaces/TextToImageAppConfig.md#properties): in a `createImageFromText()` module, assets can be passed as style reference from host application via `appConfig`.
```
Comment (nimithajalal) [inline src/pages/guides/changelog/index.md:22]: Can we add a sample here?

File: src/pages/guides/changelog/index.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ contributors:
...
+- [`CCEverywhere.appConfig.imageCompositionReference`](../../v4/shared/src/types/module/AppConfig.types/interfaces/TextToImageAppConfig.md#properties): in a `createImageFromText()` module, assets can be passed as composition reference from host application via `appConfig`.
```
Comment (nimithajalal) [inline src/pages/guides/changelog/index.md:23]: Can we add a sample here?

File: src/pages/guides/changelog/index.md [hunk 3]
```diff
@@ -15,6 +15,21 @@ contributors:
...
+
+<!-- ### Added
+
+- Support for Safari on iOS. The SDK will no longer throw an unsupported error when accessed via Safari on iOS. -->
+
+### Bug Fixes
+
+- Video Quick Action: Animate from Audio Iframe was not allowing microphone access.
```
Comment (nimithajalal) [inline src/pages/guides/changelog/index.md:31]: Let us rephrase this to, 

We have fixed the issue with Animate from Audio Iframe not allowing microphone access in Video Quick Action.

**Review by nimithajalal — CHANGES_REQUESTED** [review]

File: src/pages/guides/changelog/index.md (hunk 2 above)
Comment (undavide) [reply src/pages/guides/changelog/index.md:23]: I'm not sure it would be a good idea: we've never used the changelog as a snippet repository (I checked the add-ons, too), and given the params involved in the Embed SDK in general, it could lead to lengthy snips.
I'll rather prefer the long-term plan of adding comments and examples in the source code, and let typedoc parse and display them in the API Ref.
